routes/directories/Gamers.ts

```diff
@@ -178,35 +186,44 @@ export async function getGamerDetails(req: NextApiRequest & { params: { username
         return handleError(req, res, {message: errorObject['invalid_view']});
     }
 
+    try {
+        const gamer = await getSingleGamerData(username, platform, game_category as string);
 
-    const gamer: Gamer = await getSingleGamerData(username, platform, game_category as string);
-    let queryParams = {...allParams, uno_id: gamer.uno_id}
-    const viewToQuery: ViewQuery = getGamerDetailViewQuery(sqlView, queryParams);
+        if (!gamer) {
+            return handleError(req, res, {message: errorObject['not_found']});
+        }
 
+        const queryParams = {...allParams, uno_id: gamer.uno_id};
 
-    const classDescriptions: GamerClassDescription = await getGamerClassDescriptions();
+        const viewToQuery = getGamerDetailViewQuery(sqlView, queryParams);
 
-    if (!gamer) {
-        return handleError(req, res, {message: errorObject['not_found']});
-    }
+        await viewToQuery.executeQuery();
```

Comment:
An error was occurring here, so I had to wrap all this in a try-catch, but the real problem was that I was using the old db